### PR TITLE
RIP-295 Update submitted email text

### PR DESCRIPTION
**Pull request by lewispb**
https://eaflood.atlassian.net/browse/RIP-295

Minor tweaks to headings on the submitted confirmation email.

### CODE REVIEW

File: config/locales/flood_risk_engine/enrollment_mailer.en.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ en:
   flood_risk_engine:
     enrollment_mailer:
       submitted:
-        subject: Registration submitted
+        subject: Flood risk activity exemption submitted
+        heading_html: 'Registration submitted: <br /> please wait for us to confirm it'
```
Comment (reggieb) [inline config/locales/flood_risk_engine/enrollment_mailer.en.yml]: I don't think you should be passing markup within a translation. I'd suggest breaking this down into elements and adding the break in the view code.

File: config/locales/flood_risk_engine/enrollment_mailer.en.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ en:
...
         reference_number_heading: Your reference is %{reference_number}
-        please_wait: Please wait for us to confirm it before you start work.
         exemption_heading: Exemption submitted
         text_heading: 'Registration submitted: please wait for us to confirm it'
```
Comment (reggieb) [inline config/locales/flood_risk_engine/enrollment_mailer.en.yml]: Having text and html versions of the same text feels wrong.

Comment (lewispb) [inline config/locales/flood_risk_engine/enrollment_mailer.en.yml]: Good spot, I agree and I've removed this now.

File: config/locales/flood_risk_engine/enrollment_mailer.en.yml (hunk 1 above)
Comment (lewispb) [inline config/locales/flood_risk_engine/enrollment_mailer.en.yml]: Ok, I see what you mean. Done.

**Pull request merged**